### PR TITLE
ref(analytics): removed inbox tab clicked analytics

### PR DESCRIPTION
**Pull request by scefali**

### CODE REVIEW

File: static/app/views/issueList/header.tsx [hunk 1]
```diff
@@ -96,11 +96,7 @@ function IssueListHeader({
   function trackTabClick(tabQuery: string) {
     // Clicking on inbox tab and currently another tab is active
     if (tabQuery === Query.FOR_REVIEW && query !== Query.FOR_REVIEW) {
-      trackAnalyticsEvent({
-        eventKey: 'inbox_tab.clicked',
-        eventName: 'Clicked Inbox Tab',
-        organization_id: organization.id,
-      });
+      trackAdvancedAnalyticsEvent('inbox_tab.clicked', {organization, query: tabQuery});
```
Comment (scefali) [inline static/app/views/issueList/header.tsx:99]: adding the tab query because it's helpful

Comment (malwilley) [reply static/app/views/issueList/header.tsx:99]: Isn't `tabQuery` always the same here? `is:unresolved is:for_review assigned_or_suggested:[me, none]`

Comment (scefali) [reply static/app/views/issueList/header.tsx:99]: @malwilley well you could look at the ignored tab as well

Comment (malwilley) [reply static/app/views/issueList/header.tsx:99]: Hmm as far as I can tell this is only firing when the "For Review" tab is clicked. Doesn't look like we are tracking any of the others 🤔

Comment (scefali) [reply static/app/views/issueList/header.tsx:99]: @malwilley you're right...I'm looking at a different event. Why do we even need this event lol?

Comment (malwilley) [reply static/app/views/issueList/header.tsx:99]: Probably don't anymore, was probably used when the feature initially shipped. Doubt anyone would care if it was removed hah

File: static/app/views/issueList/header.tsx [hunk 2]
```diff
@@ -93,17 +90,6 @@ function IssueListHeader({
   const sortParam =
     queryParms.sort === IssueSortOptions.INBOX ? undefined : queryParms.sort;
 
-  function trackTabClick(tabQuery: string) {
-    // Clicking on inbox tab and currently another tab is active
-    if (tabQuery === Query.FOR_REVIEW && query !== Query.FOR_REVIEW) {
-      trackAnalyticsEvent({
```
Comment (scefali) [inline static/app/views/issueList/header.tsx:99]: @malwilley it's gone

**Review by malwilley — APPROVED** [review]

**Pull request merged**